assets/css/case-studies.scss

```
@@ -2971,6 +2971,29 @@ html:has(.cs-sticky-nav) {
         color: #fff;
       }
     }
+    p + p {
+      margin-top: 0.9rem;
+    }
+    // "ceilings" chip list (Cursor CTA) — a plain <ul> in the callout HTML
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)